src/components/donationCard/DonationCard.js

```diff
@@ -0,0 +1,160 @@
+import React from "react";
+import { useForm } from "react-hook-form";
+import ReactDom from "react-dom";
+import "../donationCard/DonationCard.scss";
+import {
+  FaHandHoldingHeart,
+  FaIdeal,
+  FaRegCalendarCheck,
+} from "react-icons/fa";
+import { GiPiggyBank } from "react-icons/gi";
+import Button from "../BtnCtA";
+import axios from "axios";
+//import { formatDistance } from "date-fns";
+
+const DonationCard = (props) => {
+  //   const createdAt = new Date(props.createdAt);
+  //   const today = new Date();
+
+  const {
+    register,
+    handleSubmit,
+    // watch,
+    // formState: { errors },
+  } = useForm();
+  const onSubmit = async (data) => {
+    console.log(data);
+    try {
+      const response = await axios.post(
+        `https://donatello-development.herokuapp.com/projects/${props.id}/donations`,
+        { donationAmount: data.amount, comment: data.comment }
+      );
+      window.location.assign(response.data.payment._links.checkout.href);
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  return ReactDom.createPortal(
+    <div className="popup">
+      <div className="donation-card">
+        <div className="donation-card__header">
```

Comment:
We need a cross icon in this header so we can close the popup (made a trello card for this)

<img width="264" alt="Screenshot 2021-07-19 at 11 22 01" src="https://user-images.githubusercontent.com/20372832/126136774-0cfcff3d-ff16-405a-b529-6a07dcd818c9.png">